src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -407,40 +504,52 @@ class Definitions {
     //  - .linkedClass: the ClassSymbol of the enumeration (class E)
     sym.owner.linkedClass.typeRef
 
-  object FunctionType {
+  object FunctionOf {
```

Comment:
Additionally, this change polluted the diff, as around 30% of lines of this PR are changed only because of those renamings.